internal/test/data/request/insert_test.go

```diff
@@ -0,0 +1,347 @@
+//
+// Copyright (C) 2019-2022 vdaas.org vald team <vald@vdaas.org>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+package request
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/vdaas/vald/apis/grpc/v1/payload"
+	"github.com/vdaas/vald/internal/errors"
+	"github.com/vdaas/vald/internal/test/comparator"
+	"github.com/vdaas/vald/internal/test/data/vector"
+	"github.com/vdaas/vald/internal/test/goleak"
+)
+
+var (
+	defaultMultiInsertReqComparators = []cmp.Option{
+		comparator.IgnoreUnexported(payload.Insert_Request{}),
+		comparator.IgnoreUnexported(payload.Insert_MultiRequest{}),
+		comparator.IgnoreUnexported(payload.Object_Vector{}),
+	}
+)
+
+func TestGenMultiInsertReq(t *testing.T) {
+	type args struct {
+		t    ObjectType
+		dist vector.Distribution
+		num  int
+		dim  int
+		cfg  *payload.Insert_Config
+	}
+	type want struct {
+		want *payload.Insert_MultiRequest
+		err  error
+	}
+	type test struct {
+		name       string
+		args       args
+		want       want
+		checkFunc  func(want, *payload.Insert_MultiRequest, error) error
+		beforeFunc func(args)
+		afterFunc  func(args)
+	}
+	dim := 10
+	comparators := append(defaultMultiInsertReqComparators, comparator.IgnoreFields(payload.Object_Vector{}, "Vector"))
```

Comment:
I think we can ignore the deepsource warning here, as it is intended usage of `append()` here.
Please let me know if you have different comment on it.